Размечаем письмо

Nitraatss opened the pull request:
Реализована базовая разметка писем.

---
:mortar_board: [Размечаем письмо](https://up.htmlacademy.ru/email/1/user/334301/tasks/3)

Nitraatss committed:
3. Размечаем письмо

Реализована базовая разметка писем.

ricomen commented on 1-invitation/index.html:
```diff
@@ -9,6 +9,234 @@
+            <table width="100%" cellspacing="0" cellpadding="0" border="0">
+              <tr>
+                <td>
+                  <p>Если вы не видите данное письмо, перейдите по <a href="#" target="_blank">ссылке</a></p>
```
Ссылкам добавить title

ricomen commented on 1-invitation/index.html:
```diff
@@ -9,6 +9,234 @@
+                  </table>
+                </td>
+                <td>
+                  <img src="./img/inv-1.png" width="352" height="286" alt="Векторный разработчик" />
```
Б15. Сбросить  border у картинок

ricomen commented on 1-invitation/index.html:
```diff
@@ -9,6 +9,234 @@
+            <table width="100%" cellspacing="0" cellpadding="0" border="0">
+              <tr>
+                <td>
+                  <p>Мы расскажем все детали о том, с чего начать свой путь в разработке сайтов, и как вырасти в востребованного специалиста. Все вопросы Вы сможете задать спикеру на вебинаре :)</p>
```
Текст прогоняем через типограф

ricomen commented on 2-welcome/index.html:
```diff
@@ -9,6 +9,243 @@
+                  </table>
+                </td>
+                <td>
+                  <img src="./img/welcome-1.png" width="175" height="269" alt="Векторный разработчик" />
```
Б20. Письмо корректно отображается на retina-дисплее

ricomen commented on 2-welcome/index.html:
```diff
@@ -9,6 +9,243 @@
+                    </tr>
+                    <tr>
+                      <td>
+                        <p><a href="mailto:study@htmlplay.com">study@htmlplay.com</a></p>
```
title

ricomen commented on 2-welcome/index.html:
```diff
@@ -9,6 +9,243 @@
+                    <tr>
+                      <td>
+                        <p><a href="mailto:study@htmlplay.com">study@htmlplay.com</a></p>
+                        <p><a href="tel:+78006907668">8-800-690-76-68</a></p>
```
title

Nitraatss committed:
Правки.

Ссылкам добавлен title.
Изображениям обнулены границы.
Добавлены изображения для ретиновых дисплеев.
Прогон текстов через https://www.artlebedev.ru/typograf/
